### PR TITLE
Deflakiness of TO IT

### PR DESCRIPTION
**Pull request by stanlyDoge**
Signed-off-by: Stanislav Knot <sknot@redhat.com>

### Type of change
- Bugfix

### Description
Affected TO was flaky... The Kafka cluster was not closed sometimes because of NPE when unmapping files (https://issues.apache.org/jira/browse/KAFKA-8398). The Adminclient was sending request to wrong instance and thus the test was timeouting. 
Using vertex instance for each test helps.

### Checklist
- [ ] Update/write design documentation in `./design`
- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: topic-operator/src/test/java/io/strimzi/operator/topic/TopicOperatorMockTest.java [hunk 1]
```diff
@@ -67,22 +65,14 @@
 
     // TODO this is all in common with TOIT, so factor out a common base class
 
-    @BeforeAll
-    public static void before() {
+    @BeforeEach
```
Comment (stanlyDoge) [inline topic-operator/src/test/java/io/strimzi/operator/topic/TopicOperatorMockTest.java:68]: To reviewers: Github diff might seem like a mess. Basically it is just moving `vertx` instantiation from `BrforeAl`l to `BeforeEach` (similarly with closing it).

File: topic-operator/src/test/java/io/strimzi/operator/topic/TopicOperatorMockTest.java [hunk 2]
```diff
@@ -138,15 +136,18 @@ public void createMockKube(VertxTestContext context) throws Exception {
     }
 
     @AfterEach
-    public void tearDown() {
+    public void tearDown(VertxTestContext context) {
+        Checkpoint checkpoint = context.checkpoint(1);
```
Comment (samuel-hawker) [inline topic-operator/src/test/java/io/strimzi/operator/topic/TopicOperatorMockTest.java:140]: Not even a suggested change, but `context.checkpoint()` calls `context.checkpoint(1)` and generally in the rest of the code we have been calling the argument-less call. But that aside this looks like a good change to me.

**Comment by stanlyDoge** [comment]
@strimzi-ci run tests

**Comment by scholzj** [comment]
Thanks fr the PR ... good fix!

**Pull request merged**